cargo-shuttle/src/lib.rs

```diff
@@ -299,6 +301,16 @@ impl Shuttle {
         Ok(())
     }
 
+    async fn clean(&self, client: &Client) -> Result<()> {
+        let lines = client.clean_project(self.ctx.project_name()).await?;
+
+        for line in lines {
```

Comment:
I'm not seeing any output after running the command, but it does clean up the `target` folder. What are we expecting these lines to be? Should we add a simple success `println`, perhaps?